org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/JDTUtils.java

```diff
@@ -433,7 +451,18 @@ private static void setPosition(Position position, int[] coords) {
 	 * @param cu
 	 * @return
 	 */
+	@Deprecated

```

Comment:
@yaohaizh @akaroml please make sure you update the debugger after the next jdt.ls release